swagger/v1-prerelease/swagger.json

```diff
@@ -0,0 +1,5792 @@
+{
+  "openapi": "3.0.1",
+  "info": {
+    "title": "API 1.0-prerelease",
+    "version": "1.0-prerelease"
+  },
+  "paths": {
+    "/v1-prerelease/changefeed": {
+      "get": {
+        "tags": [
+          "ChangeFeed"
+        ],
+        "parameters": [
+          {
+            "name": "offset",
+            "in": "query",
+            "schema": {
+              "type": "integer",
+              "format": "int64",
+              "default": 0
+            }
+          },
+          {
+            "name": "limit",
+            "in": "query",
+            "schema": {
+              "type": "integer",
+              "format": "int32",
+              "default": 10
+            }
+          },
+          {
+            "name": "includeMetadata",
+            "in": "query",
+            "schema": {
+              "type": "boolean",
+              "default": true
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "content": {
+              "text/plain": {
+                "schema": {
+                  "$ref": "#/components/schemas/JsonResult"
```

Comment:
I think things like this we'll want to change in our attributes lol. It's not very helpful to say something's schema is "JSON" haha. Perhaps that is a separate PR to clean up these schemas/contracts.

Reply:
Updated in this pr: https://github.com/microsoft/dicom-server/pull/858